src/components/App/App.tsx

```diff
@@ -188,6 +197,11 @@ export function App(props: DocInnerProps): ReactElement {
                             </PageConstructorProvider>
                         )}
                     </Page>
+                    <ConsentPopup
```

Comment:
we don't need to show <ConsentPopup when analytics not defined

Reply:
i add condition for this, but ConsentPopup components already has a check inside

Reply:
I checked locally - before you added the condition in this place, the popup was rendered regardless of the presence of the analytics block